### PR TITLE
Improvment/translation encryptable credentials

### PR DESCRIPTION
**Pull request by lkleisa**

### CODE REVIEW

File: frontend/translations/ch_be.yml [hunk 1]
```diff
@@ -118,7 +118,7 @@ ch_be:
         strong: Starch
 
     show:
-      encryptable: Encryptable
+      encryptable: Zuägangsdate
```
Comment (mtnstar) [inline frontend/translations/ch_be.yml:121]: concern sti name here.
encryptable.credentials

File: frontend/translations/ch_be.yml [hunk 2]
```diff
@@ -491,7 +491,7 @@ ch_be:
         newPassword2: "Ds nöie Passwort bestätige:"
         wrongPassword: Fautsches passwort
       encryptable:
-        encryptable: Encryptable
+        encryptable: Zuägangsdate
```
Comment (mtnstar) [inline frontend/translations/ch_be.yml:494]: does it make sense to have encryptable.encryptable ?

File: frontend/translations/en.yml [hunk 3]
```diff
@@ -490,10 +490,10 @@ en:
     settings: Settings
     profile: Profile
     add_folder: Add a new Folder
-    add_account: Add a new Encryptable
-    account_details: Show Encryptable Details
+    add_account: Add new Credentials
+    account_details: Show Credentials Details
     delete_folder: Delete the Folder
-    delete_account: Delete the Encryptable
+    delete_account: Delete the Credentials
```
Comment (mtnstar) [inline frontend/translations/en.yml:496]: rename ... the term account should not be present anymore

**Review by mtnstar — CHANGES_REQUESTED** [review]
please take some more time to clean the locale files up.

check what's still required in backend translations. we can probably remove quite a lot.
check in controllers where translation (method t) is still used. 

please concern STI encryptable names. so encryptable::credentials, encryptable::ose_secret, ...
make sure this is used everywhere in locale files. so there is no place where only encryptable is used. 

last but not least, make sure all tests are running :green_circle:

**Comment by mtnstar** [comment]
familiarize your-self with Rails i18n https://guides.rubyonrails.org/i18n.html
and emberjs https://github.com/ember-intl/ember-intl

File: config/docker/postgresql/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
+version: "2"
```
Comment (mtnstar) [inline config/docker/postgresql/docker-compose.yml:1]: this file does not belong in this PR, does it ?

File: config/docker/postgresql/postgres-prod.env [hunk 5]
```diff
@@ -0,0 +1,7 @@
+RAILS_DB_USERNAME=root
```
Comment (mtnstar) [inline config/docker/postgresql/postgres-prod.env:1]: same ...

File: config/i18n-tasks-yml [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# i18n-tasks works on itself! this is the internal config
```
Comment (mtnstar) [inline config/i18n-tasks-yml:1]: ? what is this file for?
strange extension ... -yml ?

File: config/i18n-tasks.yml [hunk 7]
```diff
@@ -0,0 +1,142 @@
+# i18n-tasks finds and manages missing and unused translations: https://github.com/glebm/i18n-tasks
```
Comment (mtnstar) [inline config/i18n-tasks.yml:1]: same file as above ... do we need it?

File: config/locales/ru.yml [hunk 8]
```diff
@@ -31,7 +31,6 @@ ru:
   maintenance_tasks: "Задачи обслуживания"
```
Comment (mtnstar) [inline config/locales/ru.yml:31]: ru still required? all other files removed?

**Review by mtnstar — CHANGES_REQUESTED** [review]
is this PR complete?
I'm a bit confused ... shouldn't there be also changes in frontend locale files?

File: frontend/translations/ch_be.yml (hunk 1 above)
Comment (lkleisa) [reply frontend/translations/ch_be.yml:80]: https://github.com/puzzle/cryptopus/issues/605

File: config/locales/en.yml [hunk 9]
```diff
@@ -7,432 +7,93 @@
...
-      updated: Folder was successfully updated.
-      deleted: Folder was successfully deleted
     file_entries:
-      deleted: Attachment was successfully deleted
-      uploaded: File was successfully uploaded.
-      uploaded_size_to_high: The file is too big to upload. (max. 10MB)
-      uploaded_filename_is_empty: The file has to be named
-      uploaded_file_inexistent: File is inexistent
-      uploaded_filename_already_exists: Filename already exists
+      uploaded: Die Datei wurde erfolgreich hochgeladen.
```
Comment (mtnstar) [inline config/locales/en.yml:79]: german ?

**Comment by mtnstar** [comment]
going to fix this in new issue #605

**Pull request closed without merging**